src/components/NewGalaxyModal.js

```diff
@@ -186,59 +198,118 @@ export const NewGalaxyModal = _.flow(
     ])
   }
 
-  const getEnvMessageBasedOnStatus = isTitle => {
-    const waitMessage = isTitle ? '' : 'This process will take up to a few minutes.'
+  const getEnvMessageBasedOnStatus = app => {
+    const waitMessage = 'This process will take up to a few minutes.'
 
     return !app ?
-      isTitle ? 'Cloud environment' : 'Environment will consist of an application and cloud compute.' :
+      'consists of application configuration, cloud compute and persistent disk(s).' :
```

Comment:
Was this change recommended by UX? Mostly curious because in the cloud env version it says `"Cloud environments consist of an application configuration, cloud compute and a persistent disk" `so this seems like a change that makes it less similar rather than more similar

Reply:
Going off of this, I will say that preference wise the two separate headings across the same sentence feels a bit off. I don't think there's actually any functional difference here, but was more of stylistic preference for me

Reply:
It wasn't particularly recommended by UX. I noticed that on the current runtime version below

![image](https://user-images.githubusercontent.com/5438223/112533684-98927f00-8d80-11eb-90be-e8f62d6d5142.png)

`Cloud environment` is duplicated and wanted to remove it. I also wanted to use text consistent across runtimes and apps, and since Galaxy apps have multiple disks, I turned `disk` into `disk(s)`.

Reply:
No strong opinions but perhaps `consists` should be capitalized?

Reply:
I can buy your rationale haha, though I agree with Rob I think capitalization would feel better. The bolded "Cloud Environment" is a title so it's feels funny to have it be like part of a fragmented sentence.

Reply:
Changed it to

-------------------------

### Cloud Environment
A cloud environment consists of application configuration, cloud compute and persistent disk(s).

-------------------------

as having a full sentence looked better to me. Let me know if any objections.